example/main.go

```diff
@@ -43,23 +50,28 @@ func listVaultsAndItems(client *onepassword.Client) {
 		}
 
 		fmt.Printf("%s %s\n", vault.ID, vault.Title)
-		items, err := client.Items.ListAll(context.Background(), vault.ID)
-		if err != nil {
+	}
+	// [developer-docs.sdk.go.list-vaults]-end
+
+	// [developer-docs.sdk.go.list-items]-start
+	items, err := client.Items.ListAll(context.Background(), vaultID)
+	if err != nil {
+		panic(err)
+	}
+	for {
+		item, err := items.Next()
+		if errors.Is(err, onepassword.ErrorIteratorDone) {
+			break
+		} else if err != nil {
 			panic(err)
 		}
-		for {
-			item, err := items.Next()
-			if errors.Is(err, onepassword.ErrorIteratorDone) {
-				break
-			} else if err != nil {
-				panic(err)
-			}
-			fmt.Printf("%s %s\n", item.ID, item.Title)
-		}
+		fmt.Printf("%s %s\n", item.ID, item.Title)
 	}
+	// [developer-docs.sdk.go.list-items]-end
```

Comment:
This should be at the end of the function, right now we are missing a brace.

Reply:
I don't think we are - the block starts within the function and ends within the function. This is what these tags produce:
```
items, err := client.Items.ListAll(context.Background(), vaultID)
if err != nil {
	panic(err)
}
for {
	item, err := items.Next()
	if errors.Is(err, onepassword.ErrorIteratorDone) {
		break
	} else if err != nil {
		panic(err)
	}
	fmt.Printf("%s %s\n", item.ID, item.Title)
}
```

Reply:
You are right.. my eyes deceived me